src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+(link: "I took the key to the Church door, but left the key to Ron’s shitty golf cart. I made a beeline for the exit")[(set: $cartKeyTaken to false)(go-to: "The Gutter - Ending")].
+
+::The Gutter - Pool table - First approach
+The teens didn’t move an inch. In fact, they seemed to be rounding first base.
+
+[I coughed.]<poolTableCough|
+[[[I turned back and let the kids have their fun|The Gutter - Inside hub]].]<poolTableExit1|
+\(click: ?poolTableCough)[(replace: ?poolTableExit1)[$dissolve[They were still ignoring me. Clothes were coming off and the only benefit was that I got to see a tattoo of a hatchet man dressed as Jesus pissing on Satan.
+
+[I kicked the leg of the goddamn table.]<poolTableKick|
+[[[I turned back. They were getting pretty close to just fucking at this point|The Gutter - Inside hub]].]<poolTableExit2|]]]
+\(click: ?poolTableKick)[(replace: ?poolTableExit2)[$dissolve["What's your fucking problem, sis?" one of the teens said, in-between tongue thrusts. "Go ruin somebody else's shit.”]]]
+
+(click: "ruin")[$dissolve[(set: $poolTableAttemptMade to true)[I frowned in a way that made me feel very old. [[Guess these horny ass kids weren’t moving for me|The Gutter - Inside hub]].]]]
+
+::The Gutter - Pool table - Second approach
+Those teens were still twisting tongues and dry humping like the world was ending. I had to respect their stamina at the very least. This wasn't something I could [[interrupt by myself|The Gutter - Inside hub]].
+
+::The Gutter - Pool table - Clear
+Though a residual funk hung over it, the table was finally clear and [[ready for a game|The Gutter - Inside hub]].
+
+::The Gutter - Jukebox - Intro
+The Jukebox was whichever punkass MC that the Honchos had decided needed to prove their salt that week. They were supposed to take requests, learn to serve the family, but most people ignored the Jukeboxes.
+
+The kid on shift today looked like he was on break. He was just fiddling with an old laptop that crunched instead of clicked. I headed his way, but my shoulders did the aw-shit slump when he turned around and I realized it was [[Donnie|The Gutter - Jukebox - Donnie]].
+
+::The Gutter - Jukebox - Donnie
+Donnie was always doing the weirdest shit. The kid barely talked, but when he did it was to quote a song. He’d get stuck on something for months at a time and couldn’t get let it go.
+
+Anna thought he was alright, but he gave me a weird vibe. Did he hate me? Did he even notice me? Impossible to tell. Still, he could have info on Anna and that was worth a lot right now.
+
+I couldn’t quite remember which song he was stuck on these days. Donnie was into all kinds of shit and I usually tried to give him a wide berth.
+
+[[“What’s up Donnie?”|The Gutter - Jukebox - Song choice]]
+
+::The Gutter - Jukebox - Song choice
+Donnie stared at his laptop. The only sound was the crunch of the trackpad and the ambient hum of the other dipshits in the bar. I had to try something else.
```

Comment:
I'm wondering how a trackpad can "crunch." What do you envision this being like?

Reply:
Nathan mentioned this as well. I guess I'm the only one who has heard a track pad crunch from getting so badly fucked up with food or liquid damage? It's pretty fucking gross and it could be a Genius Bar exclusive.

Anyway, I edited so it's all about the keys now. I think those are more common crunchers.